indicators/Adl/Adl.cs

```diff
@@ -32,7 +32,7 @@ public static partial class Indicator
                 int index = i + 1;
 
                 decimal mfm = (h.High == h.Low) ? 0 : ((h.Close - h.Low) - (h.High - h.Close)) / (h.High - h.Low);
-                decimal mfv = mfm * h.Volume;
+                decimal mfv = (h.High == h.Low) ? 0 : ((h.Close - h.Low) - ((h.High - h.Close)* h.Volume)) / (h.High - h.Low);
```

Comment:
`(h.High == h.Low) ? 0 : ((h.Close - h.Low) - ((h.High - h.Close))* h.Volume) / (h.High - h.Low)`; would make this consistent with the documented formulas; however, that would be mathematically identical to what is already there.